src/components/asset-list/RecyclerAssetList2/FastComponents/FastCoinBadge.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { View, ViewStyle } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import ArbitrumBadge from '@rainbow-me/assets/badges/arbitrumBadge.png';
+import ArbitrumBadgeDark from '@rainbow-me/assets/badges/arbitrumBadgeDark.png';
+import OptimismBadge from '@rainbow-me/assets/badges/optimismBadge.png';
+import OptimismBadgeDark from '@rainbow-me/assets/badges/optimismBadgeDark.png';
+import PolygonBadge from '@rainbow-me/assets/badges/polygonBadge.png';
+import PolygonBadgeDark from '@rainbow-me/assets/badges/polygonBadgeDark.png';
+import { AssetType } from '@rainbow-me/entities';
+
+interface FastChainBadgeProps {
+  assetType: AssetType;
+  theme: any;
+}
+
+const AssetIconsByTheme: {
+  [key in AssetType]?: {
+    dark: StaticImageData;
+    light: StaticImageData;
+  };
+} = {
+  [AssetType.arbitrum]: {
+    dark: ArbitrumBadgeDark,
+    light: ArbitrumBadge,
+  },
+  [AssetType.optimism]: {
+    dark: OptimismBadgeDark,
+    light: OptimismBadge,
+  },
+  [AssetType.polygon]: {
+    dark: PolygonBadgeDark,
+    light: PolygonBadge,
+  },
+};
+
+export const FastChainBadge = React.memo(function FastChainBadge({
+  assetType,
+  theme,
+}: FastChainBadgeProps) {
+  const { isDarkMode } = theme;
+
+  const source = AssetIconsByTheme[assetType]?.[isDarkMode ? 'dark' : 'light'];
+
+  if (!source) return null;
+
+  const imageStyles = {
+    height: 44,
+    top: 4,
+    width: 44,
+  };
+
+  const containerStyle: ViewStyle = {
+    alignItems: 'center',
+    bottom: 12,
+    elevation: 10,
+    height: 20,
+    left: -7,
+    position: 'absolute',
+    width: 20,
+    zIndex: 10,
+  };
```

Comment:
Why are these styles declared as plain objects inside of the component instead of defining them outside of a component with StyleSheet.create?